outputs.tf

```diff
@@ -6,6 +6,14 @@ output "private_subnet_ids" {
   value = ["${aws_subnet.private.*.id}"]
 }
 
+output "public_subnet_cidrs" {
+  value = ["${aws_subnet.public.*.cidr_block}"]
```

Comment:
although the module did not have descriptions in the outputs, would be useful to add descriptions - this looks bad without them https://github.com/cloudposse/terraform-aws-dynamic-subnets/pull/45/files#diff-04c6e90faac2675aa89e2176d2eec7d8R175

Reply:
Done